easysid.cpp

```diff
@@ -26,7 +37,13 @@ EXPORT void start_capture(const char *fn)
         delete wav;
         wav = NULL;
     }
-    wav = makePcmOutputWav(fn);
+    if (pcm == NULL) {
+        if (not initAudioOutput()) {
+            // TODO: Error condition
+            return;
+        }
+    }
+    wav = makePcmOutputWav(fn, pcm->getSampleRate());
```

Comment:
There is no longer a global sample rate, so it is imperative that the value be pulled from the pcm object.